File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    let answer = 0;
    let asum=0;
    let bsum=0;
    if(arr1.length>arr2.length) answer=1;
    else if(arr1.length<arr2.length) answer=-1;
    else {
        for(let i=0; i<arr1.length; i++){
            asum+=arr1[i];
        }
        for(let i=0; i<arr2.length; i++){
            bsum+=arr2[i];
        }
        if(asum>bsum) answer=1;
        else if(asum<bsum) answer=-1;
        else answer=0;
    }
    return answer;
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.cs
using System;

public class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int asum=0;
        int bsum=0;
        if(arr1.Length>arr2.Length) answer=1;
        else if(arr1.Length<arr2.Length) answer=-1;
        else {
            for(int a=0; a<arr1.Length; a++){
                asum+=arr1[a];
            }
            for(int b=0; b<arr2.Length; b++){
                bsum+=arr2[b];
            }
            if(asum>bsum) answer=1;
            else if(asum<bsum) answer=-1;
            else answer=0;
        }
        return answer;
        return answer;
    }
}